sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipant.java

```diff
@@ -187,4 +188,9 @@ public VaccinationStatus getVaccinationStatus() {
 	public void setVaccinationStatus(VaccinationStatus vaccinationStatus) {
 		this.vaccinationStatus = vaccinationStatus;
 	}
+
+	@Transient
+	public boolean isArchived() {
```

Comment:
Just double check that we really cannot archive EventParticipants?

Reply:
right now contact and evPart depend for this decision on case and event
as we wanted once the flag is added to be the single point of truth initially was thinking it is better not to add it
as at some point we need to migrate it